serializers/json/src/main/java/io/pravega/schemaregistry/serializer/json/schemas/JSONSchema.java

```diff
@@ -74,7 +74,8 @@ private JSONSchema(SchemaInfo schemaInfo, String schemaString, Class<T> derived)
      *
      * @param tClass Class whose object's schema is used.
      * @param <T> Type of the Java class. 
-     * @return {@link JSONSchema} with generic type T that extracts and captures the json schema. 
+     * @return {@link JSONSchema} with generic type T that extracts and captures the json schema.
+     * @throws IllegalArgumentException
```

Comment:
method signature needs to be changed too?

Reply:
As this method is throwing this exception, claimed by checkstylel.